src/components/search-panel/SearchPanel.tsx

```diff
@@ -191,7 +322,7 @@ const SearchPanel = () => {
 					config={{
```

Comment:
There is a lot of state here, so everything is rerendering, we can start optimizing by memoizing toggler config